### ISSUE TITLE
[Nominee] - whitelist commitment not reflected in Nominee's dashboard

### ISSUE THREAD

**Issue by glupiawymowka** [Neufund/platform-frontend#4233]
**Description**
The nominee should always have a preview of their ETO card with the current state available in the dashboard. 

However, for all the whitelist-related states there is some latency in loading the correct ETO card state. 
For example, for the whitelist full it correctly shows the number of committees, but the committed amount is 0 

**Env**
Confirmed on stage and io. 

**Steps to reproduce**
1. On .io, login as any verified investor 
2. Participate in any active whitelisting (like ISSUER_SETUP) 
3. Logout
4. Important - duplicate the tab with the dashboard
5. Login as the nominee of your whitelisted ETO (in this example, NOMINEE_SETUP)
6. Check the ETO card, refresh the page
7. Click on the card to see the preview 
8.Check the duplicated tab

**Actual results**

- [x] For ongoing whitelist, the current whitelist status is not reflected, as if nobody participated. Refresh won't help, but it will load after a couple of minutes
![image](https://user-images.githubusercontent.com/18315589/78127808-1beeee00-7415-11ea-90f7-682f642ef480.png)

- [x] For closed (not full) whitelist, the `whitelist not yet started` card is shown 
![image](https://user-images.githubusercontent.com/18315589/78127877-3a54e980-7415-11ea-9828-700e3b13756d.png)

- [x] for full whitelist, the `whitelist now open` + no slots taken state is shown 
![image.png](https://images.zenhubusercontent.com/5ae1e2334b5806bc2bd0c3bb/cff3a232-5123-41ef-a8c8-3445f49009ce)

- [x] The ETO preview also reflects the wrong (previous) state 
![image.png](https://images.zenhubusercontent.com/5ae1e2334b5806bc2bd0c3bb/95f60ff7-40c3-47f6-8fa7-7148bb3d2508)

Basically, this looks like another problem caused by whitelisting data loaded way too late. 
The correct preview sometimes shows up **in the inactive tab**  but is wrong again if you refresh the tab
![image](https://user-images.githubusercontent.com/18315589/78130262-4478e700-7419-11ea-860e-51b91f2005dd.png)

**Expected results** 
- [x] ongoing commitment reflected in nominees dashboard
- [x] whitelist closed statistics  reflected in nominees dashboard
- [x] whitelist full  statistics reflected in nominees dashboard
- [x] ETO preview is always up to date both with the ETO card and the real ETO state

**Comment by glupiawymowka** [comment]
tested on stage
